.github/workflows/translations.yml

```
@@ -0,0 +1,66 @@
+name: Update translation source
+
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every day
+    - cron: '0 0 * * *'
```

>we often do commits to master in batches anyway, I'd rather have it push those changes to transifex right away

Let's go with weekly then and try it first. We can adjust from there. I'm not really comfortable with any higher frequency.
I really feel bad for translators that get mails 3 times a week just because some PR's got merged and there are 2 strings updated each.
It's a sensitive setting that doesn't really change anything on the project side no matter if its daily or monthly considering our release schedule.

And the manual, one click run to enforce an immediate update if really needed works always.

```suggestion
on:
  workflow_dispatch:
  schedule:
    # runs every friday
    - cron: '0 0 * * 5'
  push:
    tags:
      - '*-beta*'
```